packages/zapp/console/src/components/Entities/EntityExecutions.tsx

```diff
@@ -77,7 +79,7 @@ export const EntityExecutions: React.FC<EntityExecutionsProps> = ({
   return (
     <>
       <Typography className={styles.header} variant="h6">
-        All Executions in the Workflow
+        {t('allExecutionsChartTitle', entityStrings[id.resourceType])}
```

Comment:
nit: @anrusina to weigh in here. I like going through strings to do this, however piecing strings like this is not the most localization friendly for the future. What do you think?

Reply:
Agree with Carina! As resourceType will always be in English, so when strings would be translated it would contain a mix of two languages, which is not ok. Also often when you translate sentence to a different language the words location is changed, for example "yet" word in Russian is moved between you and slept:
ENG: Have you slept **yet**?
RUS: Ты **уже** спал?

Please use patternKey for strings like this. Sample `label: t(patternKey('tableLabel', 'name')),` 
Here you will use as
```
{t(patternKey('allExecutionsChartTitle', entityStrings[id.resourceType]))}
```
In string file:
```
"allExecutionsChartTitle_": "Default string if entityStrings[id.resourceType] is null or undefined"
"allExecutionsChartTitle_Workflow": "String for Workflow"
"allExecutionsChartTitle_Tasks" : "String for Tasks"
```

Reply:
I see that you probably used as example other string, found few other instances not created by you. Will cleanup them separately :)